nx-X11/programs/Xserver/dix/atom.c

```diff
@@ -75,7 +75,7 @@ static NodePtr *nodeTable;
 void FreeAtom(NodePtr patom);
 
 Atom 
-MakeAtom(char *string, unsigned len, Bool makeit)
+MakeAtom(const char *string, unsigned len, int makeit)
```

Comment:
That doesn't seem right. `makeit` should stay `Bool` (current xserver.git master also defines it as such.)

If necessary, the value to-be-passed to it must be clipped to [0,1] first.